totalRP3/Core/Profiles.lua

```diff
@@ -627,7 +624,7 @@ function TRP3_API.profile.init()
 				showConfirmPopup(loc.PR_PROFILEMANAGER_IMPORT_WARNING:format(Utils.str.color("g") .. profiles[profileID].profileName .. "|r"), import);
 			end
 		else
-			Utils.message.displayMessage(loc.DB_IMPORT_ERROR1, 2);
+			Utils.message.displayMessage(string.format(loc.PR_IMPORT_ERROR, errorOrProfileID), 2);
```

Comment:
As a fun fact - the DB_IMPORT_ERROR1 loc string doesn't exist in core. It's an extended only thing.